general/general-02.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any, List
```

Comment:
Well done on using a set intersection, but its unnecessary to have to split such basic functionality into separate functions.
The use of typing is impressive, but probably unnecessary in this case

Reply:
Nothing actually calls any of these functions aswell